core/commonMain/src/kotlinx/serialization/modules/SerializersModule.kt

```diff
@@ -23,12 +23,28 @@ import kotlin.reflect.*
  */
 public sealed class SerializersModule {
 
+    @ExperimentalSerializationApi
+    @Deprecated(
+        "Deprecated in favor of overload with default parameter",
+        ReplaceWith("getContextual(kclass)"),
+        DeprecationLevel.HIDDEN
+    )
```

Comment:
```suggestion
    ) // Was stable since 1.0.0, HIDDEN in 1.2.0 in a backwards-compatible manner
```

Reply:
Btw, these functions (in SerializersModule, not the builder) are still marked as experimental. Should we lift this?